src/lib_sylt.rs

```diff
@@ -1,6 +1,7 @@
 use crate::*;
 use crate as sylt;
 
+#[sylt_macro::sylt_doc(dbg, "Writes the type and value of any thing you enter", [One(Value(val))] Type::Void)]
 #[sylt_macro::sylt_link(push, "sylt::lib_sylt")]
 pub fn dbg(values: &[Value], _typecheck: bool) -> Result<Value, RuntimeError> {
```

Comment:
The `push` here looks wrong. Maybe we should be able to catch it in the macro?

Reply:
That was what the "no duplicates" were supposed to catch...